staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -210,6 +279,11 @@ func (p *netadminProfile) Apply(pod *corev1.Pod, containerName string, target ru
 		useHostNamespaces(pod)
 
 	case podCopy:
+		p.removeLabels(pod)
+		p.removeAnnotations(pod)
+		p.removeProbes(pod)
+		p.removeInitContainers(pod)
+		p.removeInitContainers(pod)
```

Comment:
This is duplicate

Reply:
Thank you and I've noticed just now.
I've removed it.